app/src/index.css

```diff
@@ -97,9 +93,7 @@ li {
 }
 
 .scale-button {
-  transition: transform 0.2s ease;
-  height: 100%;
-  width: 100%;
+  @apply transition-transform duration-200 ease-in-out w-full h-full;
```

Comment:
if you changed it with tailwind, then cool, let's use it right on the spot, not in `index.css`